docs/SRS/SRS.tex

```diff
@@ -365,24 +370,182 @@ \section{Project Issues}
 
 \subsection{Open Issues}
 
+There are currently no known open issues that may lead to significant change to the product or its
+design.
+
 \subsection{Off-the-Shelf Solutions}
 
+\subsubsection{Ready-Made Products}
+
+There are existing services that solve many of the problems that this application aims to fix.
+These include AutoLeap (https://autoleap.com), Sayaaraa (https://sayaaraa.com/), and KUKUI
+(https://www.kukui.com/). These are all paid services and most offer a trial period.
```

Comment:
Is it worthwhile to mention that these ready-made products only addresses the shop management aspects? There is one solution I found which addresses the vehicle owner's needs too (https://app.openbay.com/), but i think it only serves the US market (i think)